Cargo.toml

```
@@ -1,3 +1,5 @@
+[workspace]
```

I think this might be missing
```suggestion
[workspace]
members = [ "scryer-prolog-libs" ]
```